test/unit/pages/terms.test.js

```diff
@@ -7,18 +7,20 @@
 var React = require('react');
 var TestUtils = require('react-addons-test-utils');
 var expect = chai.expect;
+var api = require('../../../app/core/api');
+var mock = require('../../../mock');
```

Comment:
You probably just added this because you thought you were gonna need it, but doesn't look like you're using it. Remove? (Minor, obviously.)